client/my-sites/media-library/list-plan-promo.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import identity from 'lodash/identity';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import VideoPressUpgradeNudge from 'blocks/upgrade-nudge-expanded';
+import { PLAN_PREMIUM, FEATURE_VIDEO_UPLOADS } from 'lib/plans/constants';
+
+export const MediaLibraryListPlanPromo = ( { translate } ) => (
+	<div className="media-library__videopress-nudge-container">
+		<VideoPressUpgradeNudge
```

Comment:
I've seen this renaming in other expanded nudge PRs so I continued that practice here for the sake of consistency, but it was strange to me too when I first noticed it. I'm renaming it to `UpgradeNudgeExpanded` here.